frontend/benefit/applicant/src/hooks/useFormActions.tsx

```diff
@@ -184,10 +197,29 @@ const useFormActions = (application: Application): FormActions => {
     return undefined;
   };
 
+  const onDelete = async (id: string): Promise<void> => {
+    try {
+      await deleteApplication(id);
```

Comment:
I probably would not use `mutateAsync` but `mutate` + `onSuccess`-callback instead: 
https://tkdodo.eu/blog/mastering-mutations-in-react-query#mutate-or-mutateasync
https://tkdodo.eu/blog/mastering-mutations-in-react-query#some-callbacks-might-not-fire


> Do UI related things like redirects or showing toast notifications in mutate callbacks. If the user navigated away from the current screen before the mutation finished, those will purposefully not fire.

Reply:
Those articles 💎 
Fixed